content/react/concepts/components/components.md

```diff
@@ -45,7 +45,20 @@ class ClassComponent extends React.Component {
 Function components are stateless and only use the `return` statement. No import statement necessary:
 
 ```jsx
-function FunctionComponent() {
+function FunctionComponent(props) {
+  return (
+    <div>
+      ...
+    <div>
+  )
+}
+```
+## Arrow function
+
+ ES6 version of JavaScript
```

Comment:
Can we turn into a complete sentence ending with a period? Maybe we could also include a hyperlink to the Docs entry for arrow functions? 

https://www.codecademy.com/resources/docs/javascript/arrow-functions